### PR TITLE
Release 2.11.2

### PR DESCRIPTION
**Pull request by Technoboy-**
### Motivation
Add docs for release 2.11.2

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: release-notes/versioned/pulsar-2.11.2.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- Fix broker restart logic [#20113](https://github.com/apache/pulsar/pull/20113)
+- Fast fix infinite HTTP call getSubscriptions caused by wrong topicName [#20131](https://github.com/apache/pulsar/pull/20131)
+- Fix getPartitionedStats miss subscription's messageAckRate [#19870](https://github.com/apache/pulsar/pull/19870)
+- Fix deadlock of zkSessionWatcher when zkConnection loss [#20122](https://github.com/apache/pulsar/pull/20122)
+- Fix Return value of getPartitionedStats doesn't contain subscription type [#20210](https://github.com/apache/pulsar/pull/20210)
+- Producer created by replicator is not displayed in topic stats [#20229](https://github.com/apache/pulsar/pull/20229)
+- Fix infinite ack of Replicator after topic is closed [#20232](https://github.com/apache/pulsar/pull/20232)
+- Skip split bundle if only one broker [#20190](https://github.com/apache/pulsar/pull/20190)
+- Fix RoaringBitmap.contains can't check value 65535 [#20176](https://github.com/apache/pulsar/pull/20176)
+- Get lowest PositionImpl from NavigableSet [#18278](https://github.com/apache/pulsar/pull/18278)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:19]:
```suggestion
- Get the lowest PositionImpl from NavigableSet [#18278](https://github.com/apache/pulsar/pull/18278)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- Fix the behavior of delayed message in Key_Shared mode [#20233](https://github.com/apache/pulsar/pull/20233)
+- Fix use error TimeUnit to record publish latency [#20074](https://github.com/apache/pulsar/pull/20074)
+- Fix the thread safety issue of BrokerData#getTimeAverageData access [#19889](https://github.com/apache/pulsar/pull/19889)
+- Fix can't send ErrorCommand when message is null value [#19899](https://github.com/apache/pulsar/pull/19899)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:23]:
```suggestion
- Fix can't send ErrorCommand when a message's value is null [#19899](https://github.com/apache/pulsar/pull/19899)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Make LedgerOffloaderFactory can load the old nar [#19913](https://github.com/apache/pulsar/pull/19913)
+- Ignore and remove the replicator cursor when the remote cluster is absent [#19972](https://github.com/apache/pulsar/pull/19972)
+- Fix the reason label of authentication metrics [#20030](https://github.com/apache/pulsar/pull/20030)
+- Optimize the ack/send future in TransactionImpl [#20271](https://github.com/apache/pulsar/pull/20271)
+- Fix NPE cause by topic publish rate limiter [#20302](https://github.com/apache/pulsar/pull/20302)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:28]:
```suggestion
- Fix NPE caused by topic publish rate limiter [#20302](https://github.com/apache/pulsar/pull/20302)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+- Fix default bundle size used while setting bookie affinity [#20250](https://github.com/apache/pulsar/pull/20250)
+- Fix ledger left in OPEN state when enable inactiveLedgerRollOverTimeMs [#20276](https://github.com/apache/pulsar/pull/20276)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:30]:
```suggestion
- Fix ledger left in OPEN state when inactiveLedgerRollOverTimeMs is enabled [#20276](https://github.com/apache/pulsar/pull/20276)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+- Fix deadlock of metadata store [#20189](https://github.com/apache/pulsar/pull/20189)
+- Fix class name typo PrecisPublishLimiter to "Precise" [#20310](https://github.com/apache/pulsar/pull/20310)
+- Fix consume stuck of shared streaming dispatcher [#18315](https://github.com/apache/pulsar/pull/18315)
+- Fix entry filter feature for the non-persistent topic [#20141](https://github.com/apache/pulsar/pull/20141)
+- Fix partitioned __change_events topic is policy topic [#20392](https://github.com/apache/pulsar/pull/20392)
+- If ledger lost, cursor mark delete position can not forward [#18620](https://github.com/apache/pulsar/pull/18620)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:36]:
```suggestion
- If a ledger is lost, the cursor mark delete position cannot forward [#18620](https://github.com/apache/pulsar/pull/18620)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+- Switch to the metadata store thread after zk operation [#20303](https://github.com/apache/pulsar/pull/20303)
+- Fix skip message API when hole messages exists [#20326](https://github.com/apache/pulsar/pull/20326)
+- Do not filter system topic while shedding [#18949](https://github.com/apache/pulsar/pull/18949)
+- Restore solution for certain topic unloading race conditions [#20527](https://github.com/apache/pulsar/pull/20527)
+- There are two same-named managed ledgers in the one broker [#18688](https://github.com/apache/pulsar/pull/18688)
+- REST Client Producer fails with TLS only [#20535](https://github.com/apache/pulsar/pull/20535)
+- Save createIfMissing in TopicLoadingContext [#19993](https://github.com/apache/pulsar/pull/19993)
+- Unwrap the completion exception [#20396](https://github.com/apache/pulsar/pull/20396)
+- Release orphan replicator after topic closed [#20582](https://github.com/apache/pulsar/pull/20582)
+- Fix the publish latency spike issue with large number of producers [#20607](https://github.com/apache/pulsar/pull/20607)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:48]:
```suggestion
- Fix the publish latency spike issue with a large number of producers [#20607](https://github.com/apache/pulsar/pull/20607)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+- [Functions] Go functions need to use static grpcPort in k8s runtime [#20404](https://github.com/apache/pulsar/pull/20404)
+- [Functions] Configure pulsar admin for TLS [#20533](https://github.com/apache/pulsar/pull/20533)
+- [Functions] Make KubernetesRuntime translate characters in function tenant, namespace, and name during function removal to avoid label errors [#19584](https://github.com/apache/pulsar/pull/19584)
+- [Functions] Exit JVM when main thread throws exception [#20689](https://github.com/apache/pulsar/pull/20689)
+- [IO Connector] Close the kafka source connector if there is uncaught exception [#20480](https://github.com/apache/pulsar/pull/20480)
+- [IO Connector] Close the kafka source connector got stuck [#20698](https://github.com/apache/pulsar/pull/20698)
+
+### Observability
+- [Broker] Fix the partitioned publisher topic stat aggregation bug [#18807](https://github.com/apache/pulsar/pull/18807)
+- [Broker] Fix topic with double quote breaks the prometheus format [#20230](https://github.com/apache/pulsar/pull/20230)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:86]:
```suggestion
- [Broker] Fix topic with double quote breaks the Prometheus format [#20230](https://github.com/apache/pulsar/pull/20230)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+- [Broker] Add JVM start time metric [#20381](https://github.com/apache/pulsar/pull/20381)
+- [Offload] Fix offload metrics error [#20366](https://github.com/apache/pulsar/pull/20366)
+
+### CLI
+- [CLI] Allow pulser-client consume create a replicated subscription [#20316](https://github.com/apache/pulsar/pull/20316)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:91]:
```suggestion
- [CLI] Allow pulser-client consumer to create a replicated subscription [#20316](https://github.com/apache/pulsar/pull/20316)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+- [WS] Remove unnecessary ping/pong implementation [#20605](https://github.com/apache/pulsar/pull/20733)
+- [Schema] Only handle exception when there has [#20730](https://github.com/apache/pulsar/pull/20730)
+
+### Library updates
+- Upgrade snakeyaml version to 2.0 [#20120](https://github.com/apache/pulsar/pull/20120)
+- Upgrade Kotlin version from 1.4.32 to 1.8.20 [#20089](https://github.com/apache/pulsar/pull/20089)
+- Upgrade the jetty server version to 9.4.51.v20230217 [#20226](https://github.com/apache/pulsar/pull/20226)
+- Upgrade sprint version to 5.3.27 [#20226](https://github.com/apache/pulsar/pull/20226)
+- Upgrade swagger version to 1.6.10 [#20226](https://github.com/apache/pulsar/pull/20226)
+- Upgrade sqlite-jdbc to resolve CVE-2023-32697 [#20411](https://github.com/apache/pulsar/pull/20411)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:112]:
```suggestion
- Upgrade SQLite JDBC to resolve CVE-2023-32697 [#20411](https://github.com/apache/pulsar/pull/20411)
```

File: release-notes/versioned/pulsar-2.11.2.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+- Upgrade Netty to 4.1.93.Final [#20423](https://github.com/apache/pulsar/pull/20423)
+- Upgrade Guava to 32.0.0 to address CVE-2023-2976 [#20459](https://github.com/apache/pulsar/pull/20459)
+- Use ubuntu 22.04 for Pulsar images [#20475](https://github.com/apache/pulsar/pull/20475)
```
Comment (momo-jun) [inline release-notes/versioned/pulsar-2.11.2.md:115]:
```suggestion
- Use Ubuntu 22.04 for Pulsar images [#20475](https://github.com/apache/pulsar/pull/20475)
```

**Review by momo-jun — COMMENTED** [review]
Hi @Technoboy-, I recommend applying the accepted updates to those PR titles so that the coming release notes won't have the same wording issues.

**Comment by Technoboy-** [comment]
> Hi @Technoboy-, I recommend applying the accepted updates to those PR titles so that the coming release notes won't have the same wording issues.

Thanks, addressed

**Review by tisonkun — APPROVED** [review]
LGTM. Thanks for preparing this!

**Review by momo-jun — APPROVED** [review]

**Pull request merged**